OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -41,65 +41,14 @@ public class CreateEffectWarhead : Warhead
 		[Desc("Chance of impact sound to play.")]
 		public readonly int ImpactSoundChance = 100;
 
-		[Desc("Consider explosion above this altitude an air explosion.",
-			"If that's the case, this warhead will consider the explosion position to have the 'Air' TargetType (in addition to any nearby actor's TargetTypes).")]
+		[Desc("Explosions above this altitude will check target validity against the 'TargetTypeAir' target types.")]
```

Comment:
We probably want to add "that don't impact an actor" between "Explosions ... above"

Reply:
Fixed.